recipes/mcmc-sampling-emcee/mcmc_sampling.ipynb

```diff
@@ -45,7 +45,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    import zeus #conda install zeus-mcmc
</pre><p>This should be in your env file, so you shouldn't need to comment it here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy-recipes/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>